data/monster/Bosses/man_in_the_cave.xml

```diff
@@ -40,7 +40,7 @@
 		<voice sentence="A MIC to rule them all!" />
 	</voices>
 	<summons maxSummons="2">
-		<summon name="Monk" interval="2000" chance="20" max="1" />
+		<summon name="Monk" interval="2000" chance="20" />
```

Comment:
Right now I'm only doing small cleanups so I'm not paying attention to these details as of now.